thali/thalilogger.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+var winston = require('winston');
+
+module.exports = function(){
+    return winston;
```

Comment:
Fixed. Here is the sample output from logcat
1439424437323 INFO app.js =>starting app.js